lib/exp.py

```diff
@@ -266,11 +269,16 @@ def to_body(self):
             'status': self.status
         }
 
-    def result(self, job_name):
+    def result(self, job):
         return Result(
-            job_name,
+            job.metadata.name,
             self.name,
-            self.uid()
+            self.uid(),
+            status={
```

Comment:
are job parameters really a result *status*?  Could see a case for separating these out and making the parameters their own top-level item.  May also want to consider referring to them as `input_parameters` or `job_parameters` to make it a bit more explicit.